exchange/hey.go

```diff
@@ -124,10 +124,7 @@ func (hs *HeyService) SendHey(ctx context.Context, pid peer.ID) error {
 	go func() {
 		defer s.Close()
 
-		err = s.SetReadDeadline(time.Now().Add(10 * time.Second))
-		if err != nil {
-			log.Error().Err(err).Msg("failed to set read deadline")
-		}
+		s.SetReadDeadline(time.Now().Add(10 * time.Second))
```

Comment:
@gallexis removed those 2 error logs as they were too noisy and made it impossible to debug anything in the tests. These methods are not that important so we don't really care it they fail. It's just a little experiment to measure latency but could probably be removed until proven useful.

Reply:
Or maybe set them as Debug() ? But yeah it's right they were too noisy